LadybugTools_Engine/Python/src/ladybugtools_toolkit/external_comfort/_externalcomfortbase.py

```diff
@@ -326,6 +328,59 @@ def plot_utci_heatmap(
             title=self.description(),
         )
 
+    def walkability_time_limits(self):
+
+        """Calculate walkability time limiits
+        Returns:
+            HourlyContinuousCollection: An object containing walkability values for each hour. 
+        """
+        csv_file = r"C:\ProgramData\BHoM\Extensions\PythonCode\LadybugTools_Toolkit\src\data\walkability.csv"
+        df = pd.read_csv(csv_file)
+        walkability_vals = []
+
+        for utci_val in self.universal_thermal_climate_index.values:
+            utci_int = int(round(utci_val,0))
+            utci_str = str(utci_int)
+
+            val = df.loc[df["UTCI temperature"] == utci_str, 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+            if utci_int < 20:
+                val = df.loc[df["UTCI temperature"] == "defaultlower", 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+            elif utci_int > 60:
+                val = df.loc[df["UTCI temperature"] == "defaultupper", 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+
+            walkability_vals.append(val[0])
+
+        header = header_from_string("Walkability Time (minutes)")
+
+        return HourlyContinuousCollection(header=header, values=walkability_vals)
+
+    def plot_walkability_heatmap( 
+            self,
+            ax: plt.Axes = None,
+        ) -> plt.Axes:
```

Comment:
```suggestion
    def plot_walkability_heatmap( 
            self,
            ax: plt.Axes = None,
            **kwargs
        ) -> plt.Axes:
```
recommend adding kwargs that will be passed to heatmap function, which can also be used to allow the user to modify some aspects of the plot (see later suggestions)